### ISSUE TITLE
"Array in array" shows wrong list of items when the cell is collapsed

### ISSUE THREAD

**Issue by ask9** [dbeaver/dbeaver#35683]
### Description

- use code:
```
SELECT
    1 AS a,
    ARRAY[ARRAY[1,2],ARRAY[5,6],ARRAY[9,10],ARRAY[11,12]] AS b
```

https://github.com/user-attachments/assets/a3e4c38b-14d1-45d3-8ed3-81e2983c2982

### DBeaver Version

Enterprise Edition 24.2.2-202409192322

### Operating System

Windows 10

### Database and driver

PostgreSQL 16
PostgreSQL JDBC Driver 42.7.4

**Comment by ShadelessFox** [comment]
Thanks for the bug report

**Comment by E1izabeth** [comment]
We fixed it in 24.2.2.
The fix is already available in Early Access version https://dbeaver.io/files/ea/
